streampark-console/streampark-console-webapp/src/api/index.js

```diff
@@ -209,6 +209,15 @@ export default {
     UPDATE: '/menu/update',
     ROUTER: '/menu/router'
   },
+  Variable: {
+    LIST: '/variable/list',
+    UPDATE: '/variable/update',
+    GET: '/variable/get',
```

Comment:
I checked,  `VariableController` doesn't have the get method.

Reply:
you are so serious，i deleted it. ^_^